twenty-first/src/math/digest.rs

```diff
@@ -416,4 +472,205 @@ pub(crate) mod digest_tests {
         let bytes_again: [u8; Digest::BYTES] = digest.into();
         prop_assert_eq!(bytes, bytes_again);
     }
+
+    // note: this test panics due to:
+    //       https://github.com/Neptune-Crypto/twenty-first/issues/195
+    //
+    //       for now we use a #[should_panic]. Depending on resolution
+    //       of #195, we may be able to remove it.
+    #[test]
+    #[should_panic]
+    pub fn bytes_in_matches_bytes_out() {
```

Comment:
nit: Is there a reason the test is `pub`? Also applies to the other tests.

Reply:
no, just habit.  That seems to be the convention in neptune-core.  I notice that Digest::test::digest_from_str() is also pub.  I will change them all to be private, for consistency.